ui-integration-test/src/main/java/co/electriccoin/zcash/ui/integration/test/screen/update/viewmodel/AppUpdateCheckerMock.kt

```diff
@@ -50,6 +51,9 @@ class AppUpdateCheckerMock private constructor() : AppUpdateChecker {
 
         val appUpdateInfoTask = fakeAppUpdateManager.appUpdateInfo
 
+        // to simulate a real-world situation
+        delay(100)
```

Comment:
I pushed a change to make this more explicit

Reply:
Oh yes, thank you!